src/sentry/search/utils.py

```diff
@@ -468,6 +468,40 @@ def _run_latest_release_query(
     return [row[0] for row in cursor.fetchall()]
 
 
+def get_terminal_release_for_group(
+    group: Group,
+    query_type: LatestReleaseOrders,
+    latest: bool,
+) -> Sequence[str]:
+    direction = "DESC" if latest else "ASC"
+    extra_conditions = ""
+    if query_type == LatestReleaseOrders.SEMVER:
+        rank_order_by = f"major {direction}, minor {direction}, patch {direction}, revision {direction}, CASE WHEN (prerelease = '') THEN 1 ELSE 0 END {direction}, prerelease {direction}, sr.id {direction}"
```

Comment:
I'm also a bit confused by the `prerelease` condition, but I assume this logic is used elsewhere

Reply:
Yeah this is just how we sort things by semver. Basically the logic is sort by `major`, `minor`, `patch`, `revision`. Then `prerelease` is either a string, meaning this is a prerelease version (alpha, beta, etc), or if it's empty then it's the real release. So empty string is considered higher, otherwise order by prerelease lexographically